src/table.rs

```diff
@@ -22,23 +22,20 @@
 // FILLED - highest index filled with live data
 //
 // Complete entry:
-// [SIZE: 2][REFS: 4][KEY: 26][VALUE: SIZE - 30]
-// SIZE: 15-bit value size. Sizes up to 0x7ffd are allowed.
+// [SIZE: 2][REFS: 4][KEY: 26][VALUE]
+// SIZE: 15-bit value size. Sizes up to 0x7ffc are allowed.
 // This includes size of REFS and KEY.
 // The first bit is reserved to indicate if compression is applied.
-// REF: 32-bit reference counter.
-// If collection is not ref counted, a single bit is used to indicate
-// if compression is needed.
-// this is removed or replaced by one byte for applied compression when need.
+// REF: 32-bit reference counter (optional).
 // KEY: lower 26 bytes of the key.
-// VALUE: SIZE-30  payload bytes.
+// VALUE: payload bytes.
 //
 // Partial entry (first part):
-// [MULTIPART: 2][NEXT: 8][REFS: 4][KEY: 26][VALUE]
-// MULTIPART - Split entry marker. 0xfffe.
+// [MULTIHEAD: 2][NEXT: 8][REFS: 4][KEY: 26][VALUE]
+// MULTIHEAD - Split entry head marker. 0xfffd.
```

Comment:
So we flag head of multipart, not sure why it is needed (for value iteration maybe, but I did  see it only for preimage and it looked like we where using the indexing for the multiparts)?

Reply:
I thought I left a comment about this, but it got lost. Yes, this is just so that the value iterator could be implemented in the future, once the new format is adopted.